feature-profile/src/main/java/cm/aptoide/pt/profile/presentation/ProfileView.kt

```diff
@@ -258,19 +255,16 @@ private fun AptoideProductsCard(openLink: (String) -> Unit) {
         onClick = { openLink("https://aptoide-uploader.pt.aptoide.com/app") }
       )
       Row(
-        modifier = Modifier
-          .fillMaxWidth()
-          .padding(start = 16.dp, top = 6.dp),
```

Comment:
Don't remove padding from this "Follow us" row, it should have 16dp start padding to match the design